src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -857,6 +939,24 @@ else if (block instanceof BlockEndPortal || block instanceof BlockMobSpawner ||
         {
             return true;
         }
+        else if(block instanceof BlockFlowerPot)
+        {
+            if (!world.setBlockState(pos, blockState, 0x03))
+            {
+                return false;
+            }
+
+            if(job instanceof JobBuilder)
+            {
+                secondaryItem = Item.getByNameOrId(((JobBuilder) job).getStructure().getBlockInfo().tileentityData.getString("Item"));
+            }
+
+            TileEntityFlowerPot tileentityflowerpot = (TileEntityFlowerPot) world.getTileEntity(pos);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'tileentityflowerpot' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)